--- jpeg.c	2012-02-01 18:24:33.000000000 +0900
+++ jpeg.c.fix	2012-02-01 18:24:39.000000000 +0900
@@ -459,7 +459,7 @@
     fprintf(stderr, "jpeg_save_tomem: No JPEG support compiled in\n");
     return 0;
 }
-int jpeg_load_from_mem(unsigned char*_data, int size, unsigned char*dest, int width, int height)
+int jpeg_load_from_mem(unsigned char*_data, int size, unsigned char**dest, int*width, int*height)
 {
     fprintf(stderr, "jpeg_load_from_mem: No JPEG support compiled in\n");
     return 0;
